_posts/2019-05-29-Student-SOCIS-2019.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Python summer with ESA and EinsteinPy
+author: sofia
+subtitle: 
+gh-repo: einsteinpy/einsteinpy
+gh-badge: [star, fork]
+tags: [News]
+comments: true
+---
+
+## Thanks to EinsteinPy and the European Space Agency
+I am so glad on being selected as a "Summer Developer" in ESA's Summer of Code in Space with EinsteinPy.
+
+First of all, I want to express my sincere gratitude to EinsteinPy and to the European Space Agency for the opportunity they gave to me.
+I am sure it is going helpful to develop my skills in computer science. So I hope my collaboration will be useful to EinsteinPy as well.
+
+##  Why EinsteinPy?
+I am truly believe that EinsteinPy project is going to be a powerful tool for researchers working in gravitational physics field. 
+The existence of simulating platforms in languages like C or C++ was a great step. 
+However, these languages are not as easy to handle for someone without a high programming skills. And even though it is usual that theoretical 
+physicist have some experience in computational areas, they shouldn't spend their energy and time in such coding process. 
+Hence, they can focus all their efforts on their theoretical physics develops.  
+
+That is why python, due to its low learning curve and clean syntax, is becoming by excellence the programming language among the scientific community.
```

Comment:
Could you replace the line with something else. It is not looking grammatically correct.